test/integration/scheduler/preemption/preemption_test.go

```diff
@@ -100,8 +100,11 @@ func waitForNominatedNodeName(cs clientset.Interface, pod *v1.Pod) error {
 const tokenFilterName = "token-filter"
 
 type tokenFilter struct {
```

Comment:
Can you add a comment describing this plugin?

Reply:
Given it's likely this PR will proceed with UT only, I will add comment in #119769.